src/userGuide/UserGuideAccordion.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactNode } from 'react';
+import { Accordion, AccordionProps } from 'hds-react';
+
+import { useMobile } from '../profile/hooks/useMobile';
+interface UserGuideAccordionProps {
+  children: ReactNode;
+  heading: string;
+  id: string;
+}
+
+function UserGuideAccordion({
+  children,
+  id,
+  heading,
+}: UserGuideAccordionProps): React.ReactElement {
+  const isMobile = useMobile();
+
+  const accordionStyle = {
+    maxWidth: '800px',
+    borderTop: 0,
+    borderBottom: 1,
+    border: 'solid 1px #ffffff',
+  };
+
+  const theme = {
+    '--background-color': '#F2F2F2',
+  };
+
+  const accordionProps: AccordionProps = {
+    style: accordionStyle,
+    size: isMobile ? 's' : 'l',
+    card: true,
+    language: 'en',
```

Comment:
Language should be parametrized with current language to have the accordion close buttons translated.

Reply:
Now it's passed